ansible_base/lib/utils/settings.py

```diff
@@ -12,7 +12,7 @@ class SettingNotSetException(Exception):
     pass
 
 
-def get_setting(name: str, default: Any = None) -> Any:
+def get_setting(name: str, default: Any = None, log_exception: bool = True) -> Any:
```

Comment:
Do we plan on using this somewhere?

Reply:
it's linked in the description, that's the only way I found, how to avoid exception loop in the early phase, before migrations - edge case